website/docs/getting-started/index.mdx

```diff
@@ -0,0 +1,558 @@
+    }
+  },
+};
+</script>
+```
+
+</MDXTab>
+
+<MDXTab label="Angular Apollo">
+
+```tsx
+import { PostsGQL, PostsQuery } from './graphql'
+//BE SURE TO USE Observable from `rxjs` and not from `@apollo/client/core` when using map
+import { Observable } from 'rxjs'
+import { map } from 'rxjs/operators'
+
+@Component({ /* ... */ })
+export class PostsComponent {
+  posts: Observable<PostsQuery['posts']>
+
+  constructor(postsGQL: PostsGQL) {
+    this.posts = postsGQL.watch().valueChanges.pipe(map(result => result.data.posts))
+  }
+}
+```
+
+</MDXTab>
+
+<MDXTab label="Svelte Apollo">
+
+```svelte
+<script lang="ts">
+  import { Posts } from "../graphql/generated";
+
+  // `posts` is fully typed, also are `Posts()` options!
+  const posts = Posts();
+</script>
+
+<ul>
+    { /* UI */ }
+</ul>
+```
+
+</MDXTab>
+
+</MDXTabs>
+
+Now, with simple configuration and an npm/yarn script, a front-end developers benefits from:
+
+
+  - **up-to-date typings**
+
+
+  - **autocompletion on all queries, mutations and, subscription variables and results**
+
+
+  - **less boilerplate** (thanks to full code generation such as React hooks generation)
+
+
+
+<br />
+
+:::tip How does GraphQL Code Generator work?
+More details on the inner working of GraphQL Code Generator are available on [this page](/docs/getting-started/how-does-it-work).
+:::
+
+
+<p>&nbsp;</p>
+
+----
+
+<p>&nbsp;</p>
+
+### To the back-end
+
+Most GraphQL API resolvers remain untyped or wrongly typed which, leads to multiple issues:
+
+- resolvers are not compliant with the schema definition
+
+- typos in the resolvers' function type signature
+
+<br/>
+
+For this reason, GraphQL Code Generator provides multiple plugins that help you automate the generation of resolvers' typings.
+
+Here are an example of a GraphQL API leveraging GraphQL Code Generator resolvers typings (based on the `schema.graphql` above):
+
+<MDXTabs namespace={`getting-started-back-end-codetabs`}>
+
+<MDXTab label="Apollo Server">
+
+```ts
+import { ApolloServer } from "apollo-server";
+import { loadDocuments } from "@graphql-tools/load";
+import { GraphQLFileLoader } from "@graphql-tools/graphql-file-loader";
+
+import { Resolvers } from './resolvers-types';
+
+
+const typeDefs = await loadDocuments('./schema.graphql', {
+    file,
+    loaders: [
+        new GraphQLFileLoader()
+    ]
+})
+
+
+const resolvers: Resolvers = {
+  Query: {
+    // typed resolvers!
+  }
+}
+
+const server = new ApolloServer({ typeDefs, resolvers });
+
+// The `listen` method launches a web server.
+server.listen().then(({ url }) => {
+  console.log(`🚀  Server ready at ${url}`);
+});
+```
+
+</MDXTab>
+
+<MDXTab label="GraphQL Yoga">
+
+```ts
+import { createServer } from '@graphql-yoga/node'
+import { loadDocuments } from "@graphql-tools/load";
+import { GraphQLFileLoader } from "@graphql-tools/graphql-file-loader";
+
+import { Resolvers } from './resolvers-types';
+
+const typeDefs = await loadDocuments('./schema.graphql', {
```

Comment:
I'd use `loadFiles` or just `readFile` from `fs` here(instead of `@graphql-tools/load` and `@graphql-tools/graphql-file-loader` packages. `loadDocuments` has a different purpose actually.